src/components/Column/ColumnBody.tsx

```diff
@@ -0,0 +1,102 @@
+import { Dispatch, SetStateAction } from "react";
+import styled, { CSSProperties } from "styled-components";
+import { DragDropContext, Droppable, DropResult } from "react-beautiful-dnd";
+
+import TicketDefinition from "../../interfaces/Ticket";
+import { DraggableTicket, Ticket } from "../Ticket/Ticket";
+
+interface ColumnBodyProps {
+  tickets: TicketDefinition[];
+  // This shouldn't be optional but we'll add API later so we can fix then I suppose.
+  onEditClick?: () => void;
+}
+
+interface DraggableColumnBodyProps extends ColumnBodyProps {
+  title: string;
+  setTickets: Dispatch<SetStateAction<TicketDefinition[]>>;
+}
+
+function getListStyle(isDraggingOver: boolean): CSSProperties {
+  return {
+    background: isDraggingOver ? "lightblue" : "inherited",
+    width: "100%",
+  };
+}
+
+export function ColumnBody({ tickets, onEditClick }: ColumnBodyProps): JSX.Element {
+  return (
+    <TicketsContainer>
+      {tickets.map((item) => (
+        <Ticket key={`ticketId:${item.id}`} title={item.name} onEditClick={onEditClick} />
+      ))}
+    </TicketsContainer>
+  );
+}
+
+export function DraggableColumnBody({
```

Comment:
It is difficult to name. This Column component should be draggable but also droppable.

Reply:
I'll change it to `DnDColumn` then, I think we both know what the term `DnD` is about